packages/cli/src/bin/cli.ts

```diff
@@ -77,6 +77,10 @@ program
     '--chromedriver-path <path>',
     'Absolute path to the desired chromedriver executable'
   )
+  .option(
+    '--chrome-path <path>',
+    'Absolute path to the desired chrome executable'
```

Comment:
With the last change we'll need to update the message as well.

```suggestion
    'Path to the desired chrome executable'
```